.github/workflows/sonar-analysis-on-merge.yml

```
@@ -0,0 +1,41 @@
+name: Sonar Analysis on Merge
```

```suggestion
name: Pull Request CI - Sonar
```
and update the filename please